src/SQLite_Command.php

```diff
@@ -79,7 +79,11 @@ public function import( $args, $assoc_args ) {
 	 * @when before_wp_load
 	 */
 	public function export( $args, $assoc_args ) {
-		WP_CLI::success( 'Exporting database...' );
+		$is_porcelain = isset( $assoc_args['porcelain'] );
+
+		if ( ! $is_porcelain ) {
+			WP_CLI::success( 'Exporting database...' );
```

Comment:
Thanks for adding the porcelain check here. I actually just realize now that i left this output when i was testing but we should probably remove it.